exercises/binary-search/src/test/java/BinarySearchTest.java

```diff
@@ -1,136 +1,128 @@
+
+        BinarySearch<Integer> sut = new BinarySearch<>(sortedListOfOddLength);
+
+        assertEquals(9, sut.indexOf(144));
     }
 
     @Ignore("Remove to run test")
     @Test
     public void findsAValueInAnArrayOfEvenLength() {
-        BinarySearch<Integer> sut
-                = new BinarySearch<>(SORTED_LIST_OF_EVEN_LENGTH);
-        final int value = 21;
-        final int actual = sut.indexOf(value);
-        final int expected = 5;
-        assertEquals(expected, actual);
+        List<Integer> sortedListOfEvenLength = Collections.unmodifiableList(
+                Arrays.asList(1, 3, 5, 8, 13, 21, 34, 55, 89, 144, 233, 377)
+        );
+
+        BinarySearch<Integer> sut = new BinarySearch<>(sortedListOfEvenLength);
+
+        assertEquals(5, sut.indexOf(21));
     }
 
     @Ignore("Remove to run test")
     @Test
     public void identifiesThatAValueIsNotIncludedInTheArray() {
-        BinarySearch<Integer> sut = new BinarySearch<>(SORTED_LIST);
-        final int value = 7;
-        final int actual = sut.indexOf(value);
-        final int expected = -1;
-        assertEquals(expected, actual);
+        List<Integer> sortedList = Collections.unmodifiableList(
+                Arrays.asList(1, 3, 4, 6, 8, 9, 11)
+        );
+
+        BinarySearch<Integer> sut = new BinarySearch<>(sortedList);
+
+        assertEquals(-1, sut.indexOf(7));
     }
 
     @Ignore("Remove to run test")
     @Test
     public void aValueSmallerThanTheArraysSmallestValueIsNotIncluded() {
-        BinarySearch<Integer> sut = new BinarySearch<>(SORTED_LIST);
-        final int value = 0;
-        final int actual = sut.indexOf(value);
-        final int expected = -1;
-        assertEquals(expected, actual);
+        List<Integer> sortedList = Collections.unmodifiableList(
+                Arrays.asList(1, 3, 4, 6, 8, 9, 11)
+        );
+
+        BinarySearch<Integer> sut = new BinarySearch<>(sortedList);
+
+        assertEquals(-1, sut.indexOf(0));
     }
 
     @Ignore("Remove to run test")
     @Test
     public void aValueLargerThanTheArraysSmallestValueIsNotIncluded() {
-        BinarySearch<Integer> sut = new BinarySearch<>(SORTED_LIST);
-        final int value = 13;
-        final int actual = sut.indexOf(value);
-        final int expected = -1;
-        assertEquals(expected, actual);
+        List<Integer> sortedList = Collections.unmodifiableList(
+                Arrays.asList(1, 3, 4, 6, 8, 9, 11)
+        );
+
+        BinarySearch<Integer> sut = new BinarySearch<>(sortedList);
+
+        assertEquals(-1, sut.indexOf(13));
     }
 
     @Ignore("Remove to run test")
     @Test
     public void nothingIsIncludedInAnEmptyArray() {
-        BinarySearch<Integer> sut = new BinarySearch<>(EMPTY_LIST);
-        final int value = 1;
-        final int actual = sut.indexOf(value);
-        final int expected = -1;
-        assertEquals(expected, actual);
+        List<Integer> emptyList = Collections.unmodifiableList(new ArrayList<Integer>(0));
```

Comment:
Maybe we could use [`Collections#emptyList`](https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#emptyList()) here?

> Returns the empty list (immutable).